### PR TITLE
#810 Review COMPILE and MODELER logs

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic [hunk 1]
```diff
@@ -19,19 +19,17 @@ StaticConnect                 =             connecting %1%_%2% with %3%
 DynamicConnect                =             connecting %1%_%2% with %3%_%4%
 ModelName                     =             model :
 ConnectedModels               =             model %1% to model %2%
-SolveParameters               =             solve Parameters of model : %1%
+SolveParameters               =             starting local initialization of model %1%
+SolveParametersOK             =             local initialization of model %1% ended successfully
```
Comment (adriguir) [inline dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic:23]: Yes!

File: dynawo/sources/Modeler/Common/DYNModeler.cpp [hunk 2]
```diff
@@ -155,15 +155,12 @@ Modeler::initParamDescription(const shared_ptr<ModelDescription>& modelDescripti
...
           int value = data_->getStaticParameterIntValue(staticID, refOrigName);
           params->createParameter(*itRef, value);
-          Trace::info("MODELER") << DYNLog(AddedRefParToSet, modelDescription->getID(), *itRef, value) << Trace::endline;
         } else if (refType == "BOOL") {
           bool value = data_->getStaticParameterBoolValue(staticID, refOrigName);
           params->createParameter(*itRef, value);
-          Trace::info("MODELER") << DYNLog(AddedRefParToSet, modelDescription->getID(), *itRef, value) << Trace::endline;
         } else {
           throw DYNError(Error::MODELER, ParameterWrongTypeReference, *itRef);
         }
```
Comment (adriguir) [inline dynawo/sources/Modeler/Common/DYNModeler.cpp:166]: I don't remember but I suppose it is handled by the Parameter log somewhere. Could you confirm?

File: dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic:23]: next MR will be about the general log, this one is only the beginning ;)

File: dynawo/sources/Modeler/Common/DYNModeler.cpp (hunk 2 above)
Comment (rosiereflo) [reply dynawo/sources/Modeler/Common/DYNModeler.cpp:166]: Not really, those are the parameters gathered in the IIDM and as you noticed earlier we mix them with the PAR ones. I plan to add again this source in another commit and then the parameters system will handle them automatically

Comment (adriguir) [reply dynawo/sources/Modeler/Common/DYNModeler.cpp:166]: Fine

**Pull request merged**